### DISCUSSION TITLE
like to contribute mermaid example

### DISCUSSION THREAD

**Discussion by wgong** [zauberzeug/nicegui#1723] [Ideas / Feature Requests]
Hi, I submitted a PR with mermaid example, please review, 
https://github.com/zauberzeug/nicegui/pull/1722

Thanks

**Comment by falkoschindler** [comment]
Thanks @wgong! We'll have a look.
I'll close this feature request since we can discuss everything related to the PR over there.

**Comment by wgong** [comment]
@falkoschindler Thanks for your feedback, I have a new PR - https://github.com/zauberzeug/nicegui/pull/1724 after removing a prior commit
